frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveExport/index.tsx

```diff
@@ -227,113 +252,143 @@ export default function SaveExport({
                 label="SVG"
+                caption="SVG CAPTION"
                 onChange={() => selectFileType("svg")}
+                checked={selectedFileTypes.includes("svg")}
                 data-testid="svg-checkbox"
               />
-            </StyledDiv>
+            </StyledInputCheckboxWrapper>
           </StyledSection>
+
           <StyledSection>
             <StyledTitle>DATA</StyledTitle>
-            <StyledDiv>
-              <StyledFormControlLabel
-                control={
-                  <StyledInputCheckBox
-                    checked={selectedFileTypes.includes("csv")}
-                  />
-                }
+            <StyledInputCheckboxWrapper>
+              <InputCheckbox
                 label="CSV"
+                caption="CSV CAPTION"
                 onChange={() => selectFileType("csv")}
+                checked={selectedFileTypes.includes("csv")}
                 data-testid="csv-checkbox"
               />
-            </StyledDiv>
+            </StyledInputCheckboxWrapper>
           </StyledSection>
-        </div>
-
-        <StyledButtonContainer>
-          <DownloadButton
-            sdsType="secondary"
-            sdsStyle="minimal"
-            onClick={handleButtonClick}
-          >
-            Cancel
-          </DownloadButton>
-          <DownloadButton
-            sdsStyle="square"
-            sdsType="primary"
-            sdsSize="large"
-            onClick={handleDownload}
-            disabled={!selectedFileTypes.length}
-            data-testid="dialog-download-button"
-          >
-            Download
-          </DownloadButton>
-        </StyledButtonContainer>
+
+          <UnderlyingDataChangeBanner width="100%" />
+
+          <StyledButtonContainer>
+            <DownloadButton
+              sdsType="secondary"
+              sdsStyle="minimal"
+              onClick={handleButtonClick}
+            >
+              Cancel
+            </DownloadButton>
+            <DownloadButton
+              sdsStyle="square"
+              sdsType="primary"
+              sdsSize="large"
+              onClick={handleDownload}
+              disabled={!selectedFileTypes.length}
+              data-testid="dialog-download-button"
+            >
+              Download
+            </DownloadButton>
+          </StyledButtonContainer>
+        </StyledModalContent>
       </StyledModal>
     </>
   );
 }
 
-function processSvg({
+function generateSvg({
   svg,
   tissueName,
-  height,
+  heatmapHeight,
+  heatmapWidth,
 }: {
   svg: string;
   tissueName: Tissue;
-  height: number;
+  heatmapHeight: number;
+  heatmapWidth: number;
 }) {
   const heatmapNode = new DOMParser().parseFromString(svg, "image/svg+xml");
   const heatmapContainer = heatmapNode
     .querySelector("foreignObject")
     ?.querySelector("div");
 
+  // This is an svg element created in dom when downloadStatus.isLoading is true
+  const banner = document.getElementById("data-message-banner");
```

Comment:
Getting the svg banner that is rendered in the UI behind the loading screen on download. This svg element will be inserted into the final svg as-is